src/Components/Nav/Nav.css

```diff
@@ -1,17 +1,19 @@
 .nav {
     background-color: #f7f7f7;
-    position: sticky;
-    top: 0;
     border-top-left-radius: 10px;
     border-top-right-radius: 10px;
+    position: sticky;
+    top: 0;
+}
+
+@media only screen and (max-device-width: 480px) {
```

Comment:
it looks like nothing happens inside this media query, so you can remove it if it's not meant to do anything